doc/nrf/libraries/others/dm.rst

```diff
@@ -17,53 +17,54 @@ The :ref:`nrfxlib:mpsl` library is used to access the radio module.
 The distance is measured during the allocated timeslot.
 
 Ranging
-*******
+=======
 
 When a new ranging request arrives, the module checks if a timeslot for the ranging can be scheduled.
 It also checks if the request fits in the schedule of rangings.
-Rangings are not executed immediately but are scheduled for future execution.
-This gives additional flexibility.
+Rangings are not executed immediately, but are scheduled for future execution, which gives additional flexibility.
 New ranging requests may arrive at any point in time.
 
 If the request can be scheduled, the module performs the ranging after the time set in the :kconfig:option:`CONFIG_DM_RANGING_OFFSET_US` option has passed.
 The ranging is executed within a timeslot.
 After ranging, a callback is called to store or process the measurement data.
 
-Using Distance Measurement
-**************************
-
-To use the Distance Measurement module, enable it using the :kconfig:option:`CONFIG_NRF_DM` Kconfig option and initialize it in your application.
-After synchronizing, the nodes that should perform the measurement with each other issue the measurement request.
-The callback function :c:func:`data_ready` is called when the measurement data is available.
+Configuration
+*************
 
-Complete the following steps:
+To use the Distance Measurement module, complete the following steps:
 
 1. Enable the :kconfig:option:`CONFIG_NRF_DM` Kconfig option.
 #. Include :file:`dm.h` in your :file:`main.c` file.
 #. Call :c:func:`dm_init()` to initialize the module.
 #. Call :c:func:`dm_request_add()` to perform the measurement.
 
-Configuration
-*************
-
-To use the Distance Measurement module, enable the :kconfig:option:`CONFIG_NRF_DM` Kconfig option.
+After synchronizing, the nodes that perform the measurement with each other will issue the measurement request.
+The callback function :c:func:`data_ready` is called when the measurement data is available.
 
-Synchronization
----------------
+Configuring synchronization
+===========================
 
 To adjust the synchronization of the nodes, change the values of the following options:
 
-* :kconfig:option:`CONFIG_DM_INITIATOR_DELAY_US` - Extra delay of start of initiator role in distance measurement. Reducing this decreases power consumption but leads to less successful rangings.
-* :kconfig:option:`CONFIG_DM_REFLECTOR_DELAY_US` - Extra delay of start of reflector role in distance measurement.
-* :kconfig:option:`CONFIG_DM_MIN_TIME_BETWEEN_TIMESLOTS_US` - Minimum time between two timeslots. This should account for processing of the ranging data after the timeslot.
+* :kconfig:option:`CONFIG_DM_INITIATOR_DELAY_US` - Extra delay of the start of the initiator role for the distance measurement.
+   Reducing this value decreases the power consumption, but leads to less successful rangings.
```

Comment:
There is some problem with spacing here: http://developer.nordicsemi.com/nRF_Connect_SDK_dev/doc/PR-7403/nrf/libraries/others/dm.html#configuring-synchronization